ui/src/views/admin/settings/SettingsView.vue

```diff
@@ -1,82 +1,145 @@
+                    >
+                      <v-text-field
+                        v-model.trim="formData.email"
+                        outlined
+                        required
+                        disabled
+                        :error-messages="errors[0]"
+                        label="E-mail"
+                        class="my-0 py-0"
+                      />
+                    </ValidationProvider>
+                  </v-col>
+                  <v-col
+                    cols="12"
+                    class="my-0 py-0"
+                  >
+                    <ValidationProvider
+                        v-slot="{ errors }"
+                        name="Password"
+                        rules="required|min:8"
+                    >
+                      <v-text-field
+                        v-model.trim="formData.password"
+                        outlined
+                        required
+                        hint="Minimum 8 characters"
+                        persistent-hint
+                        label="Password"
+                        class="my-0 py-0"
+                        :append-icon="isPasswordShown ? 'mdi-eye' : 'mdi-eye-off'"
+                        :type="isPasswordShown ? 'text' : 'password'"
+                        :error-messages="errors[0]"
+                        @click:append="isPasswordShown = !isPasswordShown"
+                      />
+                    </ValidationProvider>
+                  </v-col>
+                  <v-col
+                    cols="12"
+                  >
+                    <v-row justify="center">
+                      <v-btn
+                        color="primary"
+                        min-width="150"
+                        class="mt-2 mb-5"
+                        v-bind:disabled='!isDisabled'
+                        @click="handleSubmit(onFormSubmit)"
+                      >
+                        SAVE SETTINGS
+                      </v-btn>
+                    </v-row>
+                  </v-col>
+                </v-row>
+              </v-container>
+            </v-form>
+          </ValidationObserver>
+        </v-card>
+
+        <MaterialSnackbar
+          v-model="isSucessSnackbarShown"
+          type="success"
+        >
+          New settings have been saved!
+        </MaterialSnackbar>
       </v-col>
     </v-row>
   </v-container>
 </template>
 
 <script>
+import { mapGetters } from "vuex";
+
 export default { 
   name: "AdminSettingsView", 
   data() {
     return {
+      isFormValid: false,
+      isPasswordShown: false,
       formData: {
         email: "",
         password: ""
-      }
+      },
+      isSucessSnackbarShown: false,
     };
   },
   computed: {
-    profilePicture() {
-      return require("@assets/svg/profile_picture.svg");
-    }
-  } 
+    ...mapGetters("user", ["userData"]),
+    userEmail() {
+        return this.userData.email;
+    },
+    isDisabled() {
```

Comment:
This naming is not clear enough and doesnt explain what is happening inside the computed function (for example: isPasswordLengthOkay, isPasswordLengthSuitable, hasTypedEnoughPassword, or anything similar)